### PR TITLE
Release DAML SDK 1.14.1

### PR DESCRIPTION
**Pull request by tudor-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
lgtm assuming vmbc tests passed which I’m sure you tested, thanks!

**Comment by tudor-da** [comment]
> lgtm assuming vmbc tests passed which I’m sure you tested, thanks!

Not yet, but holding off until everything is fine there

**Comment by tudor-da** [comment]
@moritzkiefer-da Updated this PR's `LATEST` file with the SHA for 1.14.1 to the latest commit published in `release/1.14.x` (after merging https://github.com/digital-asset/daml/pull/10212)

**Comment by cocreature** [comment]
looks sensible, it’s slightly fishy to not release a separate snapshot first but given that the commit didn’t change anything in the code only build scripts, I’m fine with this but I would suggest to get @gerolf-da’s approval as well.

**Comment by gerolf-da** [comment]
I'm inclined to rather use the exact same commit for the 1.14.1 RC-snapshot and the final 1.14.1 release. There's really no gain to be had from including this commit in the 1.14.1 release itself.
If I understand #10212 correctly, it's for branch hygiene only. Let's stick to good practices for releases.

File: LATEST
Comment (gerolf-da) [inline LATEST:2]:
```suggestion
8ec16e94fff58c55858c8c97378b62cab80e1180 1.14.1
```

**Comment by cocreature** [comment]
> If I understand #10212 correctly, it's for branch hygiene only. Let's stick to good practices for releases.

It’s not, the build fails without this because we run the CI config from `main` with `build.sh` from the release branch which does not produce the log directory. The other alternative is to modify the CI config from `main` to skip the upload on releases or something like that.

**Comment by tudor-da** [comment]
> > If I understand #10212 correctly, it's for branch hygiene only. Let's stick to good practices for releases.
> 
> It’s not, the build fails without this because we run the CI config from `main` with `build.sh` from the release branch which does not produce the log directory. The other alternative is to modify the CI config from `main` to skip the upload on releases or something like that.

I can redo both releases to stand by the clean practice. Fine by you, @gerolf-da ?

File: LATEST
Comment (gerolf-da) [reply LATEST:2]: Withdrawing this suggestion, as without the commit we cannot build anymore.

**Comment by gerolf-da** [comment]
Sorry, I missed it. In this case the new snapshot also wouldn't bring much value. Let's go for 1.14.1 on the latest commit on the branch then.

**Pull request merged**
